src/components/dataset/overview/connection-to-forms.vue

```diff
@@ -39,44 +41,35 @@ except according to the terms contained in the LICENSE file.
 
 <script>
 import ExpandableRow from '../../expandable-row.vue';
+import FormLink from '../../form/link.vue';
 import I18nList from '../../i18n/list.vue';
 import SummaryItem from '../../summary-item.vue';
 
 import useRoutes from '../../../composables/routes';
+import { useRequestData } from '../../../request-data';
 
 export default {
   name: 'ConnectionToForms',
   components: {
     ExpandableRow,
+    FormLink,
     I18nList,
     SummaryItem
   },
-  props: {
-    properties: {
-      type: Array,
-      required: true
-    },
-    sourceForms: {
-      type: Array,
-      required: true
-    },
-    projectId: {
-      type: String,
-      required: true
-    }
-  },
   setup() {
+    const { dataset } = useRequestData();
     const { publishedFormPath } = useRoutes();
-    return { publishedFormPath };
+    return { dataset, publishedFormPath };
   },
   computed: {
     totalProperties() {
-      return this.properties.length;
+      return this.dataset.properties.length;
     },
     propertiesByForm() {
-      const formMap = new Map(this.sourceForms.map(f => ([f.xmlFormId, { ...f, projectId: this.projectId, properties: [] }])));
```

Comment:
We were adding `projectId` to the forms in `formMap`, but it actually didn't seem like we ever accessed `projectId` from `formMap`. We were used the `projectId` prop instead. Either way, forms will now have a `projectId` property automatically.